contracts/StorageManager.sol

```diff
@@ -16,27 +16,27 @@ contract StorageManager {
     /*
     Offer represents:
      - utilizedCapacity: how much is capacity is utilized in Offer.
-     - totalCapacity: total amount of bytes offered.
+     - totalCapacity: total amount of mega-bytes (MB) offered.
      - billingPlans: maps a billing period to a billing price. When a price is 0, the period is not offered.
      - agreementRegistry: the proposed and accepted Agreement
     */
     struct Offer {
         uint128 utilizedCapacity;
         uint128 totalCapacity;
-        mapping(uint64 => uint64) billingPlans;
+        mapping(uint64 => uint128) billingPlans;
         mapping(bytes32 => Agreement) agreementRegistry; // link to agreement that are accepted under this offer
     }
 
     /*
     Agreement represents:
      - billingPrice: price per byte that is collected per each period.
      - billingPeriod: period how often billing happens.
```

Comment:
Is this in seconds, blocks, minutes?

Reply:
Probably it is commented elsewhere, but might make sense to have the unit here too.